Sources/Hyperspace/Request/Request.swift

```diff
@@ -36,6 +36,10 @@ public protocol Request {
     /// The model type that this Request will attempt to transform Data into.
     associatedtype ResponseType
     associatedtype ErrorType: NetworkServiceFailureInitializable
+
+    /// The query parameters for the URL. These parameters should now be specified as part of the url property.
+    @available(*, deprecated: 2.0, message: "Query parameters should now be specified as part of the url property")
```

Comment:
Can we format this slightly differently: 

> Query parameters should now be specified as part of the `URL` property.

Reply:
Sure. My thought was to differentiate between the url property and the URL class. Maybe I can dig through and find cocoadoc syntax hilighting?

Reply:
Oh good point. Maybe `url` ?